docs/reference/pip_install.rst

```diff
@@ -136,23 +128,38 @@ Lastly, if you wish, you can refer to other requirements files, like this::
 Requirement Specifiers
 ++++++++++++++++++++++
 
-pip supports installing from "requirement specifiers" as implemented in
-`pkg_resources Requirements <http://packages.python.org/setuptools/pkg_resources.html#requirement-objects>`_
+pip supports installing from a package index using a :term:`requirement
+specifier <pypug:Requirement Specifier>`. Generally speaking, a requirement
+specifier is composed of a project name followed by an optional :term:`version
```

Comment:
There's a prose issue here - we say 'specifier' and 'specifiers' which on reading it make me think pip only supports a single specifier... which arguable >=1,<2 is, but lets me consistent. 'followed by optional version specifiers'.